jenkins_build.sh

```diff
@@ -2,16 +2,16 @@ RUNTIME_NAME="nodejs"
 
 CANDIDATE_NAME=`date +%Y-%m-%d_%H_%M`
 echo "CANDIDATE_NAME:${CANDIDATE_NAME}"
-export IMAGE="${DOCKER_NAMESPACE}/${RUNTIME_NAME}:${CANDIDATE_NAME}"
+IMAGE_NAME="${DOCKER_NAMESPACE}/${RUNTIME_NAME}:${CANDIDATE_NAME}"
```

Comment:
this variable needs to be exported so it can be `envsubst`ed into the `cloudbuild.yaml.in` file. also, if you're going to change the name here, you need to change it in that file as well.